platformcomponents/desktop/attachment-file-content-list.json

```diff
@@ -0,0 +1,11 @@
+{
+  "attachment-file-content-list": {
```

Comment:
I cannot find attachment-file-content-list in the component library

Reply:
Just pushed commit with links for Content tab components.
`Added Figma links for Content file uploads`
Please check again.
@nataliadelmar

Reply:
This component, named at the time of the comment `attachment-file-content-lis` has been renamed to `content-file-list-item` and has been added to the main Figma Components Catalog.
Changing this conversation to Resolved.